model-optimizer/extensions/ops/LookupTableInsert.py

```diff
@@ -0,0 +1,56 @@
+"""
+ Copyright (C) 2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import numpy as np
+
+from mo.front.common.partial_infer.utils import int64_array
+from mo.graph.graph import Node, Graph
+from mo.ops.op import Op
+
+
+class LookupTableInsertV2(Op):
```

Comment:
Why we add such an Op for V2 only?

Another question: 
Does it sound sane to create base FrameworkOp that will simply contain only op name and infer function? And then inherrit all such ops from it?

Reply:
Now it does not seem too reasonable for me since I have only model with such a case and I know only `LookupTableInsert` and `LookupTableInsertV2` operations. Based on this I can propose adequate design for `LookupTableInsert` operation from which `LookupTableInsertV2` is inherited.